Source/Core/Core/Brawlback/BrawlbackUtility.h

```diff
@@ -13,7 +13,7 @@
 #include "SlippiUtility.h"
 #include "Brawltypes.h"
 #include "Savestate.h"
-
+#include <Core/Brawlback/include/brawlback-exi-structures/ExiStructures.h>
```

Comment:
We should just have Core/Brawlback/include in the include path, then this line can become:
`#include <brawlback-exi-structures/ExiStructures.h>`